sdk/batch/Microsoft.Azure.Batch/src/Microsoft.Azure.Batch.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Description>This client library provides access to the Microsoft Azure Batch service.</Description>
     <Version>15.5.0-beta.1</Version>
-    <ApiCompatVersion>15.4.0</ApiCompatVersion>
+    <!--<ApiCompatVersion>15.4.0</ApiCompatVersion>-->
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
```

Comment:
@dpwatrous Should I revert this change? I comment out the ApiCompatVersion to pass build, but seems I shouldn't make this change.

Reply:
Reverted.